examples/class-components/0-Card-Minimal.js

```diff
@@ -22,49 +30,45 @@ class MyCheckoutForm extends React.Component {
 
   render() {
     return (
-      <ElementsConsumer>
-        {(elements) => (
-          <form
-            onSubmit={(event) => {
-              // block native form submission
-              event.preventDefault();
-
-              // Get a reference to a mounted CardElement. Elements knows how
-              // to find your CardElement because there can only ever be one of
-              // each type of element.
-              const cardElement = elements.getElement('card');
-
-              this.createPaymentMethod(cardElement);
-            }}
-          >
-            <CardElement
-              options={{
-                style: {
-                  base: {
-                    fontSize: '16px',
-                    color: '#424770',
-                    '::placeholder': {
-                      color: '#aab7c4',
-                    },
-                  },
-                  invalid: {
-                    color: '#9e2146',
-                  },
+      <form onSubmit={this.handleSubmit}>
+        <CardElement
+          options={{
+            style: {
+              base: {
+                fontSize: '16px',
+                color: '#424770',
+                '::placeholder': {
+                  color: '#aab7c4',
                 },
-              }}
-            />
-            <button type="submit">Pay</button>
-          </form>
-        )}
-      </ElementsConsumer>
+              },
+              invalid: {
+                color: '#9e2146',
+              },
+            },
+          }}
+        />
+        <button type="submit">Pay</button>
+      </form>
     );
   }
 }
 
+const InjectedCheckoutForm = () => {
```

Comment:
(super minor) `ConnectedCheckoutForm` maybe?

Reply:
I think it sounds too much like redux and might be confusing.